java-manta-it/src/test/java/com/joyent/manta/http/TCPSocketConnectionTimeoutIT.java

```diff
@@ -81,15 +103,10 @@ public void verifyConnectionTimeoutSettingWorks() throws IOException {
         Duration totalTime = Duration.between(start, stop);
 
         if (totalTime.getSeconds() != 1) {
-            String msg = String.format("Expected total connection time duration "
+            final String msg = String.format("Expected total connection time duration "
```

Comment:
I know that in general I'm very nervous about any timing based tests, but in this case, I can't see another way to test a timeout duration than to timeout and confirm the duration. Is it ok and still correct to make this `>= 1` ? Or allow some fuzz for computers and their generally terrible handling of duration? (Not that I've seen it fail.)

Reply:
Yes, it will vary for different operating systems, hence I believe this is the most viable course of action we have available to us for testing `timeouts`.